src/shared/accordion/Accordion.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import {
+  AccordionStyled,
+  AccordionText,
+  AccordionTitle,
+  AccordionTextInner,
+  Expander,
+  AccordionHeader
+} from "./Accordion.styled";
+import Typography from "../typography/Typography";
+
+const Accordion = ({ title, text }) => {
```

Comment:
Clean, vraag me alleen af wat de accesiblity hiervan is. Accesible accordion kan je vaak goed ergens vandaan plukken. Voor nu buiten scope